src/stories/Autocomplete/Component/Sections.stories.tsx

```diff
@@ -200,3 +201,32 @@ addComponentStoryDescription(
   `const args = ${stringifyWithDefaults(RenderCustomSection.args)}`,
   customSectionDescription
 );
+
+export const CustomRenderItem = ComponentTemplate.bind({});
+CustomRenderItem.args = {
+  apiKey,
+  onSubmit,
+  sections: [
+    {
+      indexSectionName: 'Products',
+      renderItem: ({ item, query }) => (
+        <div>
+          <a href={item.data?.url}>
+            <h3>{item.value}</h3>
+            <img src={item.data?.image_url} alt={item.value} />
+          </a>
+          <p>{`$ ${item.data?.price}`}</p>
+          <p>Query: {query}</p>
+        </div>
+      ),
+    },
+    {
+      indexSectionName: 'Search Suggestions',
+    },
+  ],
+};
+addComponentStoryDescription(
+  CustomRenderItem,
+  `const args = ${stringifyWithDefaults(CustomRenderItem.args)}`,
+  sectionOrderDescription
```

Comment:
```suggestion
  customRenderItemDescription
```
We also need to import it

Reply:
https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/173/files#diff-6860146af630076d83066686f7b64eac4e19f00c18f0180464a77a9777fbc7dcR17